buildSrc/src/main/java/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -108,10 +109,18 @@ public void apply(Project project) {
         TaskProvider<Copy> copyUpgradeTask = configureCopyUpgradeTask(project, upgradeVersion, upgradeDir);
         TaskProvider<Copy> copyPluginsTask = configureCopyPluginsTask(project, pluginsDir);
 
+        // Create lifecycle tasks so we can run all tests of a given type
+        Map<ElasticsearchDistribution.Type, TaskProvider<?>> lifecyleTasks = new HashMap<>();
+        lifecyleTasks.put(Type.DOCKER, project.getTasks().register("destructiveDistroTest#docker"));
```

Comment:
We've used `.` as the separator in the rest of the tasks in this plugin. Can we reuse that here? It is easier when typing out the task name to not need to add quotes as `#` requires.

Reply:
Replaced `#` with `.`.